save-demo/src/main/resources/application-mac.properties

```diff
@@ -0,0 +1 @@
+demo.fileStorage.location=/Users/Shared/.save-cloud/cnb/files
```

Comment:
BTW, what's wrong with `user.home` on Mac?

Reply:
I do not want to move existing .save_cloud from /Users/Shared to user.home